CHANGELOG.md

```diff
@@ -3,11 +3,21 @@ Changelog
 
 See also the [GitHub releases page](https://github.com/FriendsOfSymfony/FOSHttpCache/releases).
 
+2.0.0 (unreleased)
+------------------
+
+* Decoupled from the Guzzle HTTP client.
+* The NGINX purge location is no longer passed as constructor argument but by
+  calling `setPurgeLocation()`.
+* In ProxyTestCase, `getHttpClient()` has been replaced with `getHttpAdapter()`;
+  added HTTP method parameter to `getResponse()`.
+
+
 1.4.0
 -----
 
-* Added symfony/http-kernel [HttpCache client](http://foshttpcache.readthedocs.org/en/latest/proxy-clients.html#symfony-client).
-* Added [SymfonyTestCase](http://foshttpcache.readthedocs.org/en/latest/testing-your-application.html#symfonytestcase). 
+* Added symfony/http-kernel [HttpCache client](http://foshttpcache.readthedocs.org/en/stable/proxy-clients.html#symfony-client).
```

Comment:
Added 1.4.0 to the docs. If we decide to bring out more updates for 1.0, this may become cumbersome. In that case, an 1.0 branch (and building the docs for that) are preferred.